apps/web/src/app/app/[workspaceSlug]/(dashboard)/monitors/[id]/edit/page.tsx

```diff
@@ -36,15 +36,18 @@ export default async function EditPage({
       defaultSection={search.success ? search.data.section : undefined}
       defaultValues={{
         ...monitor,
+        // FIXME - Why is this not working?
+        timeout: monitor.timeout ?? 45,
+        degradedAfter: monitor.degradedAfter ?? undefined,
```

Comment:
can be removed after your fix I guess

Reply:
I keep the undefined to avoid the form error with null value